powdr/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr"
-version = "0.1.0"
-edition = "2021"
+description = "powdr crate to be used externally"
```

Comment:
```suggestion
description = "main powdr crate, reexporting the key APIs"
```

Reply:
I was confused by the description when I read it the first time. Maybe this make it better?